### DISCUSSION TITLE
Update Cruiser 5

### DISCUSSION THREAD

**Discussion by Basti32** [devemux86/cruiser#668] [Q&A] [answered]
Hallo, liebe Gemeinde,
seit dem Update von Cruiser 4 auf 5 stützt Cruiser nach kurzer Zeit bei mir immer ab. 

Früher gab es unter Info immer so eine "Infotafel" über das verwendete Gerät und Version. Diese habe ich nicht gefunden. Fand ich immer gut.
Ich nutze das Samsung Galaxy Tab Active 2
OneUi-Version 1.1 und Android Version 9
Was habe ich bis jetzt versucht: Neustart, Stopp erzwingen, alle Daten löschen und Update auf Version 5.0.1. Hat leider nichts geholfen.
Ob es ein Schema gibt, weiß ich nicht. Es schein vor allem bei erhöhter Rechenaufwand (Route ändern, navigieren) zu passieren. 

Vielleicht habt ihr noch ein paar Tipps, was ich machen kann.

Viele Grüße
Basti

Translator DeepL

Hello, dear community, since the update from Cruiser 4 to 5, Cruiser always crashes after a short time. 
Previously there was always an "info panel" under Info about the device and version used. I have not found this. 
I use the Samsung Galaxy Tab Active 2 
OneUi version 1.1 and Android version 9. 
What have I tried so far? Restart, force stop, delete all data and update to version 5.0.1. 
Unfortunately, nothing has helped. I don't know if there is a scheme. It seems to happen especially with increased computing effort (change route, navigate). 
Maybe you have a few more tips on what I can do. 
Best regards 
Basti

**Comment by devemux86** [comment]
Hi @Basti32,

You can uninstall the old version and install the new version to make sure it is a clean installation.
Google Play does not report any errors from any device. What exactly do you do and this happens?

Sie können die alte Version deinstallieren und die neue Version installieren, um sicherzustellen, dass es sich um eine saubere Installation handelt.
Google Play meldet keine Fehler von irgendeinem Gerät. Was genau tun Sie und dies geschieht?

**Comment by Basti32** [comment]
Hallo @devemux86 ,
danke für deine Hilfe.
Werde ich jetzt erst einmal machen. Wird einige Zeit dauern, bis ich alle Einstellungen wieder so habe, wie ich es möchte.
Viele Grüße
Basti

Transalator DeepL

Hi @devemux86 , 
thanks for your help, 
I'll do that for now. It will take some time until I have all the settings back the way I want them. 
best regards 
Basti

**Comment by Basti32** [comment]
Ich vergaß! 
Ich brauche ja nur mein Profil importieren und schon sind alle Einstellungen wie zuvor :-)

**Comment by devemux86** [comment]
@Basti32 

Does it work? If there is a problem, please send me your profile file to check it.

Funktioniert es? Wenn es ein Problem gibt, senden Sie mir bitte Ihre Profildatei, um sie zu überprüfen.

**Comment by Basti32** [comment]
Hallo @devemux86 ,
bis jetzt hatte ich nur noch einen Absturz. 

![Screenshot_20241107-053252_Cruiser](https://github.com/user-attachments/assets/45e4c77e-62ef-4e1a-b7ac-4f9deeff5349)
Warum jetzt diese Meldung  kam weiß ich nicht. 
Ich hatte weder GPS an, noch die Navigation. 
Habe versucht es nochmals nachzustellen, aber da lief alles normal. 
Muss ich einfach weiter beobachten.
Viele Grüße
Basti

Hello @devemux86 , 
so far I've only had one crash. 

I don't know why this message came up. I had neither GPS nor navigation on. I tried to recreate it, but everything ran normally. I'll just have to keep watching. 
Best regards 
Basti

**Comment by devemux86** [comment]
After one of the updates, there was a report on Google Play about a crash when starting the navigation.
But we need to export and send the app profile so we can check what is happening.

Nach einem der Updates gab es bei Google Play einen Bericht über einen Absturz beim Starten der Navigation.
Aber wir müssen das App-Profil exportieren und senden, damit wir überprüfen können, was passiert.

**Comment by devemux86** [comment, marked as answer]
- Fixed in https://github.com/devemux86/cruiser/discussions/641

**Comment by Basti32** [comment]
Hallo @devemux86,
habe jetzt mal wieder etwas Zeit und die App ausprobiert. Leider sind die Abstürze immer noch vorhanden. 
Habe die Version 5.1.1 noch mal deinstalliert und neu installiert ohne erkennbare Verbesserung.
Hier mal mein Profil: 
[Reiseenduro3.json](https://github.com/user-attachments/files/18285837/Reiseenduro3.json)

Würde mich freuen, wenn wir da eine Verbesserung herbeiführen können.
Viele Grüße
Basti

**Reply by devemux86** [reply]
@Basti32 

Please perform a clean installation of the application: uninstall it and reinstall it.

Bitte führen Sie eine saubere Installation der Anwendung durch: deinstallieren Sie sie und installieren Sie sie neu.

**Reply by Basti32** [reply]
Hallo @devemux86,
danke für die schnelle Antwort.

> Habe die Version 5.1.1 noch mal deinstalliert und neu installiert ohne erkennbare Verbesserung.

Dies habe ich schon getan.

Viele Grüße
Basti

**Reply by devemux86** [reply]
@Basti32 

I do not understand this. You do a fresh install of 5.1.1 and it doesn't start?
How can you export a profile then? What is your device?

Ich verstehe das nicht. Sie machen eine Neuinstallation von 5.1.1 und es startet nicht?
Wie können Sie dann ein Profil exportieren? Was ist Ihr Gerät?

**Reply by Basti32** [reply]
Hallo @devemux86,
die App stürzt nach Zeit X (5 - 20 min) ab.
Ich nutze ein Samsung Tab Active 2 (SM-T395).
Mein genutztes Profil ist 
[Reiseenduro3.json](https://github.com/user-attachments/files/18285897/Reiseenduro3.json)
Welches ich aber kurz vor der Neuinstallation exportiert habe.
Ich hoffe, das hilft Ihnen weiter.

Transaltor DeppL:


Hi @devemux86, 
the app crashes after time X (5 - 20 min). 
I'm using a Samsung Tab Active 2 (SM-T395). 
The profile I'm using is [Reiseenduro3.json](https://github.com/user-attachments/files/18285897/Reiseenduro3.json) 
which I exported just before reinstalling. I hope this helps you.

**Reply by devemux86** [reply]
@Basti32 

Please provide more details. What exactly do you do when this happens?

Bitte geben Sie mehr Details an. Was tun Sie genau, wenn das passiert?

**Reply by Basti32** [reply]
@devemux86 

Ich habe einfach nur in der Karte eine Straße gesucht. 

Transalator DeppL:

I was just looking for a street on the map.

**Reply by devemux86** [reply]
@Basti32 

This profile does not exist in Cruiser. Have you imported it?
Please do a clean installation and do not import any profiles. Then you will see how it works.

Dieses Profil existiert nicht in Cruiser. Haben Sie es importiert?
Bitte machen Sie eine saubere Installation und importieren Sie keine Profile. Dann sehen Sie, wie es funktioniert.

**Comment by Basti32** [comment]
@devemux86,

>  Bitte machen Sie eine saubere Installation und importieren Sie keine Profile. Dann sehen Sie, wie es funktioniert.

Dann hätte ich ja das Problem, das ich alle Einstellungen wieder neue machen muss. Dafür sind doch diese Profile da?

Transalator DeppL:

&gt; Please do a clean installation and do not import any profiles. Then you will see how it works.

Then I would have the problem that I would have to change all the settings again. Isn't that what these profiles are for?

**Reply by devemux86** [reply]
@Basti32 

In the reports, we must first understand how this happens. Otherwise we can't improve it.
So we need the detailed steps of how something happens.

In den Berichten müssen wir zunächst verstehen, wie das geschieht. Sonst können wir ihn nicht verbessern.
Wir brauchen also die detaillierten Schritte, wie etwas passiert.

**Reply by devemux86** [reply]
@Basti32

In your steps above, there is no mention of importing a profile. We need the detailed steps.

In Ihren obigen Schritten wird nicht erwähnt, dass Sie ein Profil importieren. Wir brauchen die detaillierten Schritte.

**Comment by Basti32** [comment]
@devemux86,
habe die App neu installiert.
Änderungen der Einstellungen:
![Screenshot_20250101-172245_Cruiser](https://github.com/user-attachments/assets/918a6183-3646-4ea8-9815-70a18f97a252)
![Screenshot_20250101-172255_Cruiser](https://github.com/user-attachments/assets/5f111e6c-5d52-4c42-8ef1-7f7757b6aff3)
Habe dann die Offline-Karte Mapsforge Germany ausgewählt.
Maximal 5 min genutzt vor Absturz.

Transalator DeppL:
I have reinstalled the app. 
Changes to the settings: 
I then selected the offline map Mapsforge Germany.
Used for a maximum of 5 minutes before crashing.

**Reply by devemux86** [reply]
@Basti32 

You are using an SD card for the storage folder? The SD card may have a problem.
Use the standard internal memory or try another SD card.

Sie verwenden eine SD-Karte für den Speicherordner? Vielleicht hat die SD-Karte ein Problem.
Verwenden Sie den standardmäßigen internen Speicher oder versuchen Sie es mit einer anderen SD-Karte.

**Reply by devemux86** [reply]
@Basti32 

> Habe dann die Offline-Karte Mapsforge Germany ausgewählt.

Do you mean downloading the map with the app's download manager?

Meinen Sie den Download der Karte mit dem Download-Manager der App?

**Comment by Basti32** [comment]
@devemux86,
werde es ausprobieren. Dauert aber etwas.

Transalator DeppL

will try it out. But it will take a while.

**Comment by devemux86** [comment]
@Basti32 

If I understand your steps correctly, are you doing a clean install and trying to download an offline map? You need to leave the download screen open until the download is complete. Then you can use the app.

Wenn ich deine Schritte richtig verstehe, machst du eine saubere Installation und versuchst, eine Offline-Karte herunterzuladen? Sie müssen den Download-Bildschirm geöffnet lassen, bis der Download abgeschlossen ist. Dann können Sie die App verwenden.

**Comment by Basti32** [comment]
Hallo @devemux86,
so ist es. Nebenbei überprüfe ich noch meine SD-Karte. Mein interner Speicher ist leider nicht groß genug, um mehrere Karten zu speichern, deswegen brauche ich die SD-Karte. Alles Schritte zusammen dauern etwas.
Viele Grüße
Basti

Transaltor DeepL:

Hello @devemux86, 
that's right. I'm also checking my SD card. Unfortunately, my internal memory is not big enough to store several cards, so I need the SD card. All steps together take some time. 
best regards 
Basti

**Comment by Basti32** [comment]
Hallo @devemux86,
leider haben die Maßnahmen Neuinstallation und Download-Karte auf internen Speicher keine Besserung gebracht. Es hat schon gereicht, 5 min in der Karte zu zoomen und die Kartenausschnitte zu verändern, bis Cruiser abgestürzt ist. 
Als Profil hatte ich diesmal Auto.
Zum Profil Reiseenduro 3: Es ist nur das Profil Motorrad mit meinen Änderungen der Einstellungen von Cruiser. Es ist kein eigens von mir geschriebenes Profil. Ich hoffe, sie verstehen, was ich damit ausdrücken will.

Viele Grüße 
Basti

Translator DeepL:

Hi @devemux86, 
unfortunately reinstalling and downloading the map to the internal memory did not help. It was enough to zoom in on the map for 5 minutes and change the map sections until Cruiser crashed. I had Auto as the profile this time. 

Regarding the Reiseenduro 3 profile: It is only the motorbike profile with my changes to the Cruiser settings. It is not a profile I wrote myself. I hope you understand what I'm trying to say. 

best regards 
Basti

**Reply by devemux86** [reply]
@Basti32 

Since Android 7 is very old, I tried an Android 7 emulator and everything seems to work correctly.
Maybe you can try another device? Also try [Cruiser 5.2.0 (Beta)](https://github.com/devemux86/cruiser/discussions/690).

The test was done with a clean installation and the download of an offline map.
Have other users experienced a problem?

---

Da Android 7 sehr alt ist, habe ich einen Android 7-Emulator ausprobiert und alles scheint korrekt zu funktionieren.
Vielleicht können Sie ein anderes Gerät versuchen? Versuchen Sie auch [Cruiser 5.2.0 (Beta)](https://github.com/devemux86/cruiser/discussions/690).

Der Test wurde mit einer sauberen Installation und dem Download einer Offline-Karte durchgeführt.
Haben andere Benutzer ein Problem festgestellt?

**Reply by MotoUKRider** [reply]
I have a [Samsung Galaxy Tab A](https://www.gsmarena.com/samsung_galaxy_tab_a_7_0_(2016)-7880.php) from 2016 running 5.1.1 - the 8GB storage isn't enough (even with little else installed) for Cruiser and a few maps so I run an 8GB micro-SD card but Cruiser still requires the "device" memory to perform the download, I guess an Android thing/bug/quirk/feature!

I gave it a spin last week and the original installation/profile worked OK as did my latest profiles from my main device. Apart from being a little sluggish when moving around the map and calculating a particularly "fun" route, there's no other issues that I've encountered. I cannot recall it ever crashing, and it's always been "installed to SD Card".

**Reply by devemux86** [reply]
> requires the "device" memory to perform the download, I guess an Android thing/bug/quirk/feature!

@MotoUKRider This is correct. Such functions only work in internal storage.

**Comment by Basti32** [comment]
Hallo @devemux86,
würde gerne die Cruiser Beta 5.2.0 probieren, doch wo bekomme ich diese her?
Auf meinen dem Tab ist aber Android 9 installiert.

Hi @devemux86, 
I would like to try the Cruiser Beta 5.2.0, but where can I get it? 
Android 9 is installed on my tab.

Hallo @MotoUKRider,
ich benutze gerne einen Simulator, um meine Karten auf dem PC zu downloaden und dann in die App zu integrieren.

Hi @MotoUKRider, 
I like to use a simulator to download my maps on the PC and then integrate them into the app.

best regards
Basti

**Reply by WolfgangZ42** [reply]
> I would like to try the Cruiser Beta 5.2.0, but where can I get it?

https://github.com/devemux86/cruiser/discussions/690#discussioncomment-11636831

**Reply by Basti32** [reply]
Hallo @WolfgangZ42 ,
glaube ich habe es gefunden. 
Viele Grüße
Basti

Hello @WolfgangZ42 , 
I think I have found it. 
best regards 
Basti

**Comment by Basti32** [comment]
Hallo,
bis jetzt läuft Cruiser 5.2.0 Beta gut.
Was haltet ihr für kritischer, den Speicherort auf SD Karte ändern oder mein Profil mit meinen Einstellungen improtieren und aktivieren?

Viele Grüße
Basti

Transaltor DeepL
Hi, 
so far Cruiser 5.2.0 Beta is running fine. 
What do you think is more critical, change the location to SD card or improvise and activate my profile with my settings? 
Best regards 
Basti

**Comment by devemux86** [comment]
On devices with small internal storage, you can download offline maps manually with the browser, save them to the SD card and open them in Cruiser via the "Map | External maps" menu. First activate the extended menu in the app's advanced settings.

Auf Geräten mit kleinem internen Speicher können Sie Offline-Karten manuell über den Browser herunterladen, auf der SD-Karte speichern und in Cruiser über das Menü „Karte | Externe Karten“ öffnen. Aktivieren Sie dazu zunächst das erweiterte Menü in den erweiterten Einstellungen der App.

**Comment by Basti32** [comment]
Hallo @devemux86,
danke für den Tipp.
Bei den externen Karten habe ich das Problem, dass ich nur eine Karte aktivieren kann.
Kann eine defekte Landkarte die Ursache für die Abstürze sein? 
Beim Kopieren der Daten mit Überprüfung gab es Problem mit der Italy-Karte.

Habe nun den Test der SD-Karte abgeschlossen und es wurde kein Fehler festgestellt.
Zur Zeit nutze ich Cruiser 5.2.0 ohne Probleme. In den Einstellungen habe ich den Speicherort wieder auf die SD-Karte geändert. 
Dazu habe ich noch eine Frage: Warum muss ich meine Daten beim Ändern des Speicherortes immer im alten Ordner löschen? 
Wäre doch schön, wenn ich aus Gründen X das nicht machen musste.
![Screenshot_20250104-060139_Cruiser](https://github.com/user-attachments/assets/189a9b8f-722c-45f3-9437-f82aaddd95f8)

Als Profil nutze ich noch Motorrad. Da ich vorab schon einmal mein Profil Reiseenduro 3 genutzt habe, sind jetzt die Menüfarben wieder blau, nicht wie nach der Neuinstallation (Gelb). Da stellt sich mir die Frage, welche Einstellungen werden durch das Profil gespeichert/geändert?
Gibt es dazu eine Dokumentation?
Viele Grüße
Basti
PS: Um diese Nachricht kurz zu halten, habe ich mal auf eine Übersetzung verzichtet.

**Reply by devemux86** [reply]
@Basti32 

You can select multiple files to open them as external maps.
All settings are saved in the profiles if you have changed them.

Sie können mehrere Dateien auswählen, um sie als externe Maps zu öffnen.
Alle Einstellungen werden in den Profilen gespeichert, sofern Sie sie geändert haben.

**Comment by Basti32** [comment]
Hallo @devemux86,
in der Version 4 kam bei externen Karten immer die Option Karte hinzufügen. Diese Option habe ich scheinbar nicht mehr, deswegen wird bei mir immer nur die aktuell ausgewählte Karte angezeigt.
Aber das ist für mich nicht weiter schlimm. Es geht scheinbar jetzt und das reicht mir.
Nochmals einen großen Dank 👍 an sie und auch @WolfgangZ42 , @MotoUKRider  für die gute Hilfe bei meinem Problem.
Viele Grüße
Basti

**Reply by devemux86** [reply]
@Basti32 

First activate the extended menu in the app's advanced settings.

Aktivieren Sie dazu zunächst das erweiterte Menü in den erweiterten Einstellungen der App.

**Reply by Basti32** [reply]
Hallo @devemux86,
wie sie weiter oben sehen, habe ich dies schon getan.
Viele Grüße
Basti

Translator DeepL:

Hello @devemux86, 
as you can see above, I have already done this. 
best regards 
Basti

**Comment by Basti32** [comment]
Hallo @devemux86 ,

> Sie können mehrere Dateien auswählen, um sie als externe Maps zu öffnen.
Alle Einstellungen werden in den Profilen gespeichert, sofern Sie sie geändert haben.

Um eine bessere Ordnung bei meinen Karten zu bekommen, habe ich die Karten in Länder-Ordner sortiert. Somit konnte ich nicht gleichzeitig auf mehrere Länder zu zugreifen. Da ich die Option hinzufügen (Cruiser 4xx) nicht gesehen habe, bin ich davon ausgegangen, dass diese bei Cruiser 5 nicht mehr geht.
Viele Grüße
Basti

> You can select multiple files to open them as external maps.
All settings are saved in the profiles if you have changed them.

Transaltor DeepL:

To keep my maps better organised, I sorted them into country folders. This meant that I could not access several countries at the same time. As I didn't see the add option (Cruiser 4xx), I assumed that this was no longer possible with Cruiser 5. 
Best regards 
Basti

**Comment by Basti32** [comment]
Zurück zur Themenüberschrift:
Mir ist gerade noch eine Möglichkeit für die Abstürze von Cruiser 5 eingefallen. Um internen Speicherplatz zu sparen, ändere ich, wenn möglich, den Speicherort von den APPs gerne zum externen Speicher. Könnte da eine mögliche Ursache liegen?

Wie schon mal geschrieben, es geht jetzt, und das reicht mir! Wollte nur auch diese Möglichkeit mit einbringen. Da sie ja nicht sehen können, was ich bis dato mit Cruiser 5 gemacht habe.

Transaltor DeepL:

Back to the topic title: 
I have just thought of another possibility for the Cruiser 5 crashes. To save internal storage space, I like to change the storage location of the apps to external storage whenever possible. Could this be a possible cause? 

As I've written before, it works now and that's enough for me! I just wanted to add this possibility. As they can't see what I've done with Cruiser 5 so far.

**Comment by Basti32** [comment]
Hallo,
leider stützt Cruiser 5.2.1 wieder ab. 
Habe Cruiser wieder deinstalliert und Version 5.2.1 direkt installiert. Nur leider ohne Erfolg. Kann wieder kein Muster erkennen, warum Cruiser einfach abstürzt. 
Habe jetzt mal Routen von mehr als 600 km erstellen lassen, zwecks größerer Rechenoperation. Daran scheint es auch nicht zu liegen.
Habe Kartenordner auf der SD Karte (ist laut Schreibtest ok) 
Profil ist Auto (Standard)
Es besteht kein Abo.
Wäre dankbar für weitere Tipps, was ich noch probieren kann. 
Viele Grüße
Basti
PS: Kann ich zu Cruiser 4 zurück gehen?

Transaltor DeepL

Hello, 
unfortunately Cruiser 5.2.1 is crashing again. I uninstalled Cruiser again and installed version 5.2.1 directly. 
Unfortunately without success. Once again, I can't recognise any pattern as to why Cruiser simply crashes. 
I have now had routes of more than 600 km created for the purpose of a larger calculation operation. 
That doesn't seem to be the problem either. 
I have map folders on the SD card (according to the write test this is ok) 
Profile is Auto (default) 
There is no subscription. 
Would be grateful for any other tips on what else I can try. 
Best regards 
Basti
PS: Can I go back to Cruiser 4?

**Reply by devemux86** [reply]
@Basti32 

Have you tried a different phone? And do not use an SD card.
It works fine for everyone else, so maybe it's your specific device?

Haben Sie ein anderes Telefon ausprobiert? Und verwenden Sie keine SD-Karte.
Bei allen anderen Benutzern funktioniert es einwandfrei, vielleicht liegt es also an Ihrem speziellen Gerät?

**Reply by devemux86** [reply]
@Basti32 

Next Cruiser will be released soon and you will also be able to test it on this device.

Der nächste Cruiser wird bald erscheinen und Sie werden ihn auch auf diesem Gerät testen können.

**Comment by Basti32** [comment]
Hallo @devemux86 ,
danke für ihre Antwort. 
Ich muss eine SD-Karte für die Karten verwenden, da mein Telefonspeicher nicht reicht.
Habe jetzt ein Zweitgerät im Test. Nur leider hat mein Tab Android 9 und das Zweitgerät Android 11. 
Beim Zweitgerät scheint es keine Abstürze zu geben. Das hilft mir aber jetzt nicht weiter. Da ich das Tab für die Navigation verwende. Interessant ist eben, dass diese Abstürze es seit Cruiser 5 aufgetreten. Deswegen meine Frage zwecks Cruiser 4. Kurviger Pro (1.14.28) funktioniert tadellos auf beiden Geräten. Und da ist die App sogar auch auf der SD-Karte und benutzt die gleichen Karten.
Es muss ja nicht zwingend an Cruiser liegen. Darum suche ich ja nach Vorschlägen, was ich probieren kann. 
Die SD-Karte hat einen kompletten Speichertest (Schreiben/ Lesen) ohne Fehler bestanden.
Steht also noch die Frage, was kann ich probieren.
Viele Grüße
Basti

Translator DeepL:

Hi @devemux86 , 
thanks for your reply. 
I have to use an SD card for the maps as my phone memory is not enough. I now have a second device on test. Unfortunately, my tab has Android 9 and the second device has Android 11. 
There don't seem to be any crashes on the second device. But that doesn't help me now. As I use the tab for navigation. The interesting thing is that these crashes have occurred since Cruiser 5. Hence my question about Cruiser 4. Kurviger Pro (1.14.28) works perfectly on both devices. And the app is even on the SD card and uses the same maps. It doesn't necessarily have to be Cruiser. That's why I'm looking for suggestions on what I can try. 
The SD card has passed a complete memory test (write/read) without any errors. 
So the question remains, what can I try? 
Best regards 
Basti

**Reply by devemux86** [reply]
@Basti32 

Please wait for the next Cruiser. It will have the previous map engine again.

Bitte warten Sie auf den nächsten Cruiser. Es wird wieder die vorherige Karte Motor haben.

**Comment by Basti32** [comment]
Hallo @devemux86,
danke für ihre Antwort. 
Habe zwar noch einen Verdacht/Möglichkeit, werde aber warten bis zum nächsten Cruiser :-)
Viele Grüße
Basti.

Translator DeepL

Hello @devemux86, 
thank you for your reply. 
I still have a suspicion/possibility, but will wait until the next cruiser :-) 
Best regards 
Basti.
